controller.go

```diff
@@ -169,19 +177,19 @@ func usage() {
 }
 
 func main() {
-	log.Printf("starting %s", os.Args[0])
+	log.Infof("starting %s", os.Args[0])
 	var (
 		awsAdapter  *aws.Adapter
 		kubeAdapter *kubernetes.Adapter
 		kubeConfig  *kubernetes.Config
 		err         error
 	)
 	if err = loadSettings(); err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
This should Fatal, because of restarting automatically and trying to get AWS credentials from cache again